### PR TITLE
Refs #21238 - Translate setting tooltip

### PR DESCRIPTION
**Pull request by tbrisker**
Also display default for array type settings.

### CODE REVIEW

File: app/models/setting.rb [hunk 1]
```diff
@@ -286,7 +286,7 @@ def self.column_check(opts)
   # End methods for loading default settings
 
   def full_name_with_default
-    "#{full_name} (Default: #{default})"
+    _("%{full_name} (Default: %{default})") % {full_name: _(full_name), default: default}
```
Comment (akofink) [inline app/models/setting.rb:289]: do you need to have the translation for `_(full_name)` if the whole thing is translated?

**Review by akofink — COMMENTED** [review]
Seems fine to me. Just one question.

File: app/models/setting.rb (hunk 1 above)
Comment (tbrisker) [reply app/models/setting.rb:289]: yes, the full_name is extracted separately ins the setting definition. the translation for the whole string is actually just to get the default translated in context correctly

**Review by akofink — APPROVED** [review]
APT!

**Review by boaz0 — APPROVED** [review]
LGTM

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Merged, thanks @tbrisker!

And thanks @akofink & @ripcurld0.

**Pull request merged**